packages/next-templates/package.json

```diff
@@ -36,6 +36,7 @@
     "@utrecht/design-tokens": "1.0.0-alpha.607",
     "@utrecht/web-component-library-react": "1.0.2",
     "@utrecht/web-component-library-stencil": "1.1.0",
-    "clsx": "2.0.0"
+    "clsx": "2.0.0",
+    "prettier": "3.2.5"
```

Comment:
Waarom moeten we `prettier` installeren? Is die niet al onderdeel van de workspace `package.json`?